webapp/middleware/authenticated.js

```diff
@@ -1,4 +1,5 @@
 import isEmpty from 'lodash/isEmpty'
```

Comment:
It could be interessting to write some tests for the redirect.

Reply:
If it’s okay then I like to merge this PR and have it in my mind for the next time I touch it, because I needed half a day of restarting the test to get it through.